roles/ipabackup/tasks/get_ipabackup_dir.yml

```diff
@@ -1,6 +1,6 @@
 ---
 - name: Get IPA_BACKUP_DIR dir from ipaplatform
-  command: "{{ ansible_playbook_python }}"
+  command: "{{ ansible_python_interpreter | default('/usr/bin/python') }}"
```

Comment:
I don't see a problem with it, but see PR #517.
If we stop assuming injected facts, this will have to change.

Reply:
Right ,I could fix it right now as the ansible_facts dict is more general but I agree that this should be consistent across the code base. So keep me posted when you have a decision, quick fix